CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contributing
+
+The Flux Terraform provider is [Apache 2.0
+licensed](https://github.com/fluxcd/flux2/blob/main/LICENSE) and
+accepts contributions via GitHub pull requests. This document outlines
+some of the conventions on to make it easier to get your contribution
+accepted.
+
+We gratefully welcome improvements to issues and documentation as well as to
+code.
+
+## Certificate of Origin
+
+By contributing to this project you agree to the Developer Certificate of
+Origin (DCO). This document was created by the Linux Kernel community and is a
+simple statement that you, as a contributor, have the legal right to make the
+contribution. No action from you is required, but it's a good idea to see the
```

Comment:
Sorry I missed this in the previews comment round, but this paragraph is no longer true, as we require signed-off commits.

Reply:
Check now please